schemas/User.yaml

```diff
@@ -3,12 +3,53 @@ type: object
 required:
   - username
   - password
+  - email
```

Comment:
this property is required but has been deleted from the properties below.
please just remove `email` form the required properties